netlify/constants.ts

```diff
@@ -73,4 +73,12 @@ export const SUPPORTED_NETWORKS: supportedNetworks = {
     ...SUPPORTED_CHAINS[CHAIN_ID.xdcapothem],
     provider: jsonRpcProvider(SUPPORTED_CHAINS[CHAIN_ID.xdcapothem].rpcUrl),
   },
+  [CHAIN_ID.HederaMainnet]: {
+    ...SUPPORTED_CHAINS[CHAIN_ID.HederaMainnet],
+    provider: jsonRpcProvider(SUPPORTED_CHAINS[CHAIN_ID.HederaMainnet].rpcUrl),
+  },
+  [CHAIN_ID.HederaTestnet]: {
+    ...SUPPORTED_CHAINS[CHAIN_ID.HederaTestnet],
+    provider: jsonRpcProvider(SUPPORTED_CHAINS[CHAIN_ID.HederaTestnet].rpcUrl),
+  },
```

Comment:
Please make sure that u are updating the package.json file as i dont think this object exist in the version of tradetrust-utils. please update it to at lease v1.9.1, thanks!

Reply:
added